### PR TITLE
refactor(l10n): Activate fxa-react rebrand strings, update header logo

### PR DESCRIPTION
**Pull request by LZoog**
Because:
* We want to activate our new Mozilla branding strings and logo

This commit:
* Uses new strings, removes old strings
* Adds mozilla 'm' logo SVG, references it in relevant files

closes FXA-8323

---

Slight overlap with FXA 8322 / FXA 7983. See [previous fxa-react PR](https://github.com/mozilla/fxa/pull/15800/files) (`app-logo-alt-3` was added in another PR).

I added the "m" logo as a new svgo'd SVG that'll be cached after initial network request since we'll use this in the banner, header, and on the Settings and Backbone "card", [see our note about inline vs non-inline](https://github.com/mozilla/fxa/tree/main/packages/fxa-settings#non-inlined-svgs). If we prefer inline here since the SVG is very small, that's fine with me too, but I'll probably update it to use the inline method outlined there for React and we may want the SVG to use as a background image anyway for the "card".

I can update the favicon with Settings / content-server changes.

image
image

### CODE REVIEW

**Review by vbudhram — APPROVED** [review]
@LZoog 
giphy

File: packages/fxa-react/components/LogoLockup/index.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 import React, { ReactElement } from 'react';
 import { Localized } from '@fluent/react';
-import logo from '../../images/ff-logo.svg';
+import logo from '../../images/moz-m-logo.svg';
```
Comment (vbudhram) [inline packages/fxa-react/components/LogoLockup/index.tsx:7]: Does this need to be optmized?

Comment (LZoog) [reply packages/fxa-react/components/LogoLockup/index.tsx:7]: Already ran it through SVGO😎

**Review by dschom — APPROVED** [review]

**Review by bcolsson — APPROVED** [review]

**Comment by LZoog** [comment]
I pushed an update to revert the FTL file changes, see description in #15914 as to why.

**Pull request merged**